### ISSUE TITLE
content: CO[OE]P

### ISSUE THREAD

**Issue by kaycebasques** [GoogleChrome/web.dev#2328]
* Authors
  * @agektmr    
* Deadline: 2020 April 10 (**Hard**)
* Drafts
  * https://docs.google.com/document/d/1AJoa-Ggp_sv8I_NzFSYemS6l8_hLHBtM2JWopDWmZ_k/edit
* Stakeholders
  * @jpmedley  
  * @rowan-m

**Comment by jpmedley** [comment]
The beta blog is posted on March 26. For anything that has to be live before then, I've been asking folks to have done by March 25.

**Comment by agektmr** [comment]
As an FYI,
the URL is coordinated with DevTools team in advance since they wanted to embed one in a console message.
It's web.dev/coop-coep .
I believe this shouldn't be an issue?

**Comment by jpmedley** [comment]
Why are there two drafts?

**Comment by agektmr** [comment]
I just put them in the content proposal form because I was asked "Add a link to your a draft of your doc and URLs to any related content that's already published."
The first draft link is the one I'm working on.

**Comment by jpmedley** [comment]
If the second one is not part of the proposal, can you please remove it.

**Comment by agektmr** [comment]
Done.

**Comment by agektmr** [comment]
Hi, we are getting closer to full LGTM from reviewers. At which point should I request a tech writer review? In the Google Doc state or pull request state?

**Comment by jpmedley** [comment]
Request a tech writer review next.

**Comment by kaycebasques** [comment]
@agektmr I see that the technical reviews are done and just confirmed with @jpmedley that he'll review the Google Doc today (it'll be more efficient to let him review the Google Doc today before submitting a PR)

**Comment by jpmedley** [comment]
@kaycebasques I chatted with Eiji last evening. There are still some expert reviewers who haven't signed off. We agreed to give them until the EOB on Wednesday. I will edit this on Thursday.

**Comment by agektmr** [comment]
Sorry @jpmedley , but thinking about the branch cut on Thursday, I'd like to extend the deadline by 24 hours - EOD on Thursday. That way the team can determine what they can land in 83 for sure.

**Comment by agektmr** [comment]
I have submitted a new content proposal, but there's a new article span-off from this one since this article becomes way too long than I originally thought. The draft can be found here: https://docs.google.com/document/d/1bprwXaQM6ZD5K8iEMnx-n-iKpNeCTSp3pabRor2YUVI/edit#

**Comment by kaycebasques** [comment]
@agektmr regarding your [last comment](https://github.com/GoogleChrome/web.dev/issues/2328#issuecomment-607053720) I like how you handled it. Submitting the content proposal form again (which led me to create a separate issue). This is good because it helps us properly report all of the work that's being done.

Regarding the current status of this post, it looks like it's just waiting on a TW review from @jpmedley.

**Comment by agektmr** [comment]
We still have the other half of the content to publish. Requires @jpmedley to review so I can start converting it to markdown. Reopening.

**Comment by kaycebasques** [comment]
@agektmr what's the other related content? Please link me to a draft or PR. I see that Joe is reviewing, I just want to stay in the loop and have access to the draft later if needed.

**Comment by agektmr** [comment]
It's in [the same doc](https://docs.google.com/document/d/1AJoa-Ggp_sv8I_NzFSYemS6l8_hLHBtM2JWopDWmZ_k/edit). We've decided to split this draft into two separate articles.
